src/App.js

```diff
@@ -1,21 +1,16 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
```

Comment:
always prefer absolute imports to be added before relative imports.

Reply:
ok